src/views/LoginView.vue

```diff
@@ -268,69 +260,80 @@ export default {
       this.showSnackBar = false;
       this.snackBarMessage = '';
     },
-   isAlphaNumeric(val){
-      return (/^[\w.@-]+$/).test(val) || 'Username must contain only letters or numbers';
+    isAlphaNumeric(val) {
+      return /^[\w.@-]+$/.test(val) || 'Username must contain only letters or numbers';
     },
-    has8OrMoreCharacters(val){
+    has8OrMoreCharacters(val) {
       return val.length >= 8 || 'Password must contain at least eight characters';
     },
+    scrollAndFocusLogin() {
+      this.$refs.loginContainer.scrollIntoView();
+      this.$refs.usernameInput.focus();
+    },
+    directToSignup() {
+      this.$router.push({ name: ROUTE_NAME_SIGNUP });
+      this.scrollAndFocusLogin();
+    }
```

Comment:
I feel like we could make this one function but I'm unsure if a boolean or string value is better.

```     scrollAndFocusLogin(signup) {
      if (signup) {
        this.$router.push({ name: ROUTE_NAME_SIGNUP });
      }
      this.$refs.loginContainer.scrollIntoView();
      this.$refs.usernameInput.focus();
    }, ```

Reply:
Actually I think the button in the top right of the page should jump you to the form without changing the url i.e. if you're on the login page it should keep you on login and if you're on the signup page it should keep you on signup.

In that case I think we don't need the conditional logic here at all, just scroll the form into view and focus the username